roseWong/assignments/twopointers/lc1/lc1.py

```diff
@@ -1,37 +1,48 @@
 # Problem Statement #
-# Given an array of sorted numbers and a target sum, find a pair in the array whose sum is equal to the given target.
-
-# Write a function to return the indices of the two numbers (i.e. the pair) such that they add up to the given target.
+# Given an array of sorted numbers and a target sum,\
+# find a pair in the array whose sum is equal to the given target.
+
+# Write a function to return the indices of the two numbers (i.e. the pair)
+# such that they add up to the given target.
+# Psuedocode
+""" 
+if empty array, return empty array
+initiate left pointer at index zero and right pointer at index array length - 1
+sum left pointer and right pointer
```

Comment:
This can be reworded to "sum the values at the left pointer and right pointer." It will make it clearer to your interviewer.